test/TesterInternal/StreamingTests/SQSStreamTests.cs

```diff
@@ -8,62 +8,59 @@
 using UnitTests.StorageTests.AWSUtils;
 using UnitTests.StreamingTests;
 using Xunit;
+using Orleans.Runtime.Configuration;
 
 namespace UnitTests.Streaming
 {
     [TestCategory("AWS"), TestCategory("SQS")]
     public class SQSStreamTests : HostedTestClusterPerTest
     {
-        internal static readonly FileInfo SiloConfigFile = new FileInfo("Config_SQSStreamProviders.xml");
-        internal static readonly FileInfo ClientConfigFile = new FileInfo("ClientConfig_SQSStreamProviders.xml");
         public static readonly string SQS_STREAM_PROVIDER_NAME = "SQSProvider";
 
-        private static readonly TestingSiloOptions sqsSiloOptions = new TestingSiloOptions
-        {
-            SiloConfigFile = SiloConfigFile,
-        };
-        private static readonly TestingClientOptions sqsClientOptions = new TestingClientOptions
-        {
-            ClientConfigFile = ClientConfigFile
-        };
-
         private SingleStreamTestRunner runner;
 
-        public override TestingSiloHost CreateSiloHost()
+        public override TestCluster CreateTestCluster()
         {
+            var options = new TestClusterOptions(initialSilosCount: 4);
```

Comment:
This is similar to the previous PR, where 4 nodes are defined in the xml config.  Shall we just move the TestCluster defaults for 2?